pkg/registry/core/service/storage/rest.go

```diff
@@ -125,6 +126,15 @@ var (
 	_ rest.StorageVersionProvider = &REST{}
 )
 
+// getServiceIPs gets ip allocator object suited for the service on basis of its associated network
+func (rs *REST) getServiceIPs(service *api.Service) (ipallocator.Interface, error) {
```

Comment:
I have a bit concern about the naming. Here the function does not return the IPs, instead it returns the allocator. Is it better to rename the func as getServiceIPAlloc?

Reply:
point made. will change the method name as suggested.